mintapi/api.py

```diff
@@ -194,26 +194,13 @@ def get_session_cookies(self, username, password):
         while not driver.current_url.startswith('https://mint.intuit.com/overview.event'):
             time.sleep(1)
 
-        # get ius_session cookie by going to accounts.intuit.com
+        # Get session cookies by going to accounts.intuit.com
         driver.get("http://accounts.intuit.com")
```

Comment:
While testing this, I'm noticing that this request no longer provides the data we would expect.  It simply renders the login form again regardless of the success/failure of the login.

Reply:
Right; I'm not really sure I follow the background behind the change (see https://github.com/mrooney/mintapi/commit/85892cfb7d9b2b5a9d112825fb789d683e1d7161#diff-4117f091b0b8c6d18c2469a6f2452c55) to have this additional nav before grabbing cookies.

https://mint.intuit.com/overview.event should have all the cookies already at this point. Not sure where/how a different domain would address this.

My suggestion would be to remove this extra "redirect", unless I'm missing context. @dherg

Reply:
Upon further review, it looks like the ius_session cookie is only after the accounts.intuit.com visit. I was trying to trace the new auth flow with inspector. It seems that the value of ius_session is still sent out to pf.intuit.com/../clear.png, despite never being sent to the client in that form. My guess is there is an alternative representation for the ius_session cookie.

I'm curious why you're not getting the ius_session cookie. Maybe a sleep or some sort is required before grabbing the cookies after the accounts.intuit.com visit?